one-ui/src/main/java/com/marklogic/hub/curation/services/FlowManagerService.java

```diff
@@ -100,6 +99,30 @@ public void deleteFlow(String flowName) {
         flowManager.deleteFlow(flowName);
     }
 
+    public List<Flow> getFlows() {
```

Comment:
Shouldn't this method be on FlowManager? FlowManager has knowledge of how to read files and get flow names, so I think it should know how to return a list of flows as well (and sort them too).